### DISCUSSION TITLE
Doesn't work

### DISCUSSION THREAD

**Discussion by SergioXII12** [angristan/wireguard-install#206] [Help]
I tried both clean installation on CentOS8 and Debian 10 (server side), and Windows 10 and iOS 14.2 on client side.
No success. The windows wireguard client establish connection but without any internet access. Same for iOS client.
The installation script executed without any troubles.

**Comment by Saugjunkie** [comment]
did you a reboot after install wireguard?

**Comment by SergioXII12** [comment]
Definitely yes !

**Comment by quyleanh** [comment]
On your server, is there any config about firewall and iptables?

**Comment by SergioXII12** [comment]
No, just clean fresh installations on both distros. 
No firewall (fully disabled). Script looks very logical simple and easy to use. So I'm frustrated where the troubles could be.
I completely understand how wireguard works and how to install it manually and I have working installation on one of my servers. But I like this script in terms of new clients adding and I cannot make it working.

**Comment by ah1102** [comment]
Working for Debian 10: https://github.com/ah1102/wireguard-install/blob/master/wireguard-install.sh

_need add: apt-get install wireguard-dkms wireguard-tools linux-headers-$(uname -r)_

**Comment by ghost** [comment]
same issues as the author - i have tried a few clean installations with my vps and this doesn't work.  
Could Not Resolve Host

**Comment by Minc3** [comment]
Hello all,
It sounds like you need to edit /etc/sysctl.conf and uncomment #net.ipv4.ip_forward=1

sudo nano /etc/sysctl.conf
Remove # from #net.ipv4.ip_forward=1
Optional: ipv6: #net.ipv6.conf.all.forwarding=1
Then restart your system.

Let me know if this helped.

**Reply by SergioXII12** [reply]
I did it. Same result

**Comment by quyleanh** [comment]
Check your network resolve.
Try to ping to your client and server IP.
Then let see if it works.

**Reply by SergioXII12** [reply]
Of course all is pingable from both locations.

**Reply by quyleanh** [reply]
How about your firewall?
